pages/index.tsx

```diff
@@ -52,7 +53,9 @@ const Home: NextPage = () => {
   useEffect(() => {
     if (input !== '') {
       const stringArray = input.split('/');
```

Comment:
and what in the name of the allmighty is inside stringarray? what are its contents? idk idk idk